.github/workflows/msi.yaml

```diff
@@ -0,0 +1,31 @@
+name: Build the MSI (Installer)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
So for now, until we figure out CI properly, I'd avoid building the installer for each commit. Eventually we can have CI build the installer and add it to releases artifacts, or push it to winget.

Can you change this workflow to run on manual triggers only for now?